eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -37,7 +37,11 @@ jobs:
       - publish: $(Build.ArtifactStagingDirectory)/Build/Products/Release-iphonesimulator
         displayName: 'Publish release artifacts [iphonesimulator]'
         artifact: 'bundles-release-iphonesimulator'
-
+      
+      - script: |
+          swift build
+        displayName: "Use swift build for SPM"
+  
```

Comment:
Not sure if this script is enough or needs more details. Or needs to be in its own job, currently I just put it under the existing build job

Reply:
It should be sufficient. If you exclude one of your exclusions does it fail the CI?

Reply:
Created commit b469331 where I removed some files and you can see the CI failed for that case so looks like it is working as expected.